test/functional/vimscript/system_spec.lua

```diff
@@ -630,15 +630,18 @@ end)
 describe('shell :!', function()
   before_each(clear)
 
-  it(':{range}! with powershell filter/redirect #16271', function()
+  it(':{range}! with powershell filter/redirect #16271 #19250', function()
     local screen = Screen.new(500, 8)
     screen:attach()
     local found = helpers.set_shell_powershell(true)
+
     insert([[
       3
       1
       4
       2]])
+
+    -- when command has no argument
     feed(':4verbose %!sort<cr>')
```

Comment:
just change this one, we don't need a separate test case

Reply:
Oh, right.

Done.